### DISCUSSION TITLE
Coppa link outdated?

### DISCUSSION THREAD

**Discussion by Alex-e107nl** [e107inc/e107#5117] [General]
Hi when i click on the info link in the signup coppa text [https://www.ftc.gov/documents/0493-Complying-with-COPPA-Frequently-Asked-Questions](https://www.ftc.gov/documents/0493-Complying-with-COPPA-Frequently-Asked-Questions) i get an Page not found....

Can it be changed? To what?

**Comment by Moc** [comment]
Reported as bug and fixed: https://github.com/e107inc/e107/issues/5121. Thanks!
